it_hardware/__manifest__.py

```diff
@@ -0,0 +1,72 @@
+{
+    'name': 'IT Hardware Sales and Support',
+    'version': '1.0',
+    'category': 'Services',
+    'description': """
+We offer IT sales, installation, and repair services for a wide range of products, including laptops, RAM, accessories, and CCTV cameras.
+""",
+    'depends': [
+        'account_followup',
+        'helpdesk_stock',
+        'knowledge',
+        'product_expiry',
```

Comment:
```suggestion
```

Reply:
It will cause issue as this [https://shorturl.at/ZRZNQ](url) record comes from the above dependency

Reply:
Yes, but then we need to change the xpath to a field that comes from stock and not product_expiry